db/cart_items.js

```diff
@@ -84,10 +84,25 @@ async function deleteCartItems(id) {
     }
 }
 
+async function deleteCart(userId) {
+    try{
+        const { rows: [ cart_item ] } =   await client.query(`
```

Comment:
I think you want just { rows } here instead of "rows: [ cart_item ]" since we're deleting multiple rows from the table.